api/job.proto

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+syntax = "proto3";
+package openpitrix;
+
+// set go package name to pb
+option go_package = "pb";
+
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+	info: {
+		title: "OpenPireix Project";
+		version: "0.0.1";
+		contact: {
+			name: "OpenPireix Project";
+			url: "https://openpitrix.io";
+		};
+	};
+	schemes: HTTP;
+	consumes: "application/json";
+	produces: "application/json";
+};
+
+
+message CreateJobRequest {
+	string _ = 1; // placeholder for job_id
+	string cluster_id = 2;
+	string app_id = 3;
+	string app_version = 4;
+	string job_action = 5;
+	string directive = 8;
+}
+
+message CreateJobResponse {
+	string job_id = 1;
+	string cluster_id = 2;
+	string app_id = 3;
+	string app_version = 4;
+}
+
+message Job {
+	google.protobuf.StringValue job_id = 1;
```

Comment:
IMO, the empty job_id is invalid, why doe not use `string job_id = 1`?

PS: proto3's zero value is import feature, it is designed to fit Go's zero value.

Reply:
In order to unify the Response, we will all the `string` in the Response writing `google.protobuf.StringValue`